### DISCUSSION TITLE
How to set an app's availability to use ABM (Apple Business Manager)?

### DISCUSSION THREAD

**Discussion by squallstar** [fastlane/fastlane#17073] [General]
I am looking to automate the process of the "**Pricing and Availability**" section in App Store Connect using Fastlane and I cannot find any option to change the distribution type to "**Available for private distribution to specific organizations on Apple Business Manager or Apple School Manager**" under the "**Pricing and Availability > Distribution for Business and Education**" section.

**This tells Apple that the app is intended to be distributed via ABM instead of via the App Store.**

Reference screenshot:

![image](https://user-images.githubusercontent.com/574210/90640683-94254700-e230-11ea-9b3f-7e7b932f9fdb.png)

Notes in regarding to the fields in the UI:

- Organization ID and Organization Name are found by logging in to business.apple.com and go to Settings > Enrolment information.
- After the codes are redeemed, an email will be sent to you with the access to a spreadsheet containing all the links. The redemption code table appeared after I bought, but the download link took a while to appear. 

---

Can Fastlane support this type of app distribution and if so, how can it be set when using `deliver`?

cc @joshdholtz

**Comment by squallstar** [comment]
@joshdholtz is there any update on the above? Many thanks.

**Comment by tonytlwu** [comment]
@joshdholtz Is this a feature that would likely be added soon?

**Comment by max-ott** [comment]
Please don't spam fastlane team members with `@` multiple times – this is just rude. `fastlane` is still an open source project. Feel free to create a pull request or search for a contractor doing this for you.

**Comment by joshdholtz** [comment]
A fix for this is in my queue but a few other high important issues have come up since then so it got pushed down a little 😔 Hopefully have this out sooner than later.

**Reply by joshdholtz** [reply]
The issue is right here - https://github.com/fastlane/fastlane/issues/17244
